test/init.js

```diff
@@ -7,8 +7,8 @@ var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
 global.getConfig = function (options) {
 
   var dbConf = {
-    host: config.host || 'localhost',
-    port: config.port || 3306,
+    host: process.env.MYSQL_HOST || config.host || 'localhost',
+    port: process.env.MYSQL_PORT || config.port || 3306,
     database: 'myapp_test',
```

Comment:
@rmg the database name is used by the tests so we'll just have to run those builds sequentially if they conflict. I'll add the username and password